<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(88, 253, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_iniciar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>169</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Presiona el boton</string>
    </property>
    <property name="statusTip">
     <string>Presiona el boton</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>INICIAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Tiempo:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pausar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>269</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="toolTip">
     <string>Presiona el boton</string>
    </property>
    <property name="statusTip">
     <string>Presiona el boton</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>PAUSAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reiniciar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>219</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Presiona el boton</string>
    </property>
    <property name="statusTip">
     <string>Presiona el boton</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>REINICIAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_min">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_seg">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_marcas">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>369</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Presiona el boton</string>
    </property>
    <property name="statusTip">
     <string>Presiona el boton</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>MARCAS</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>281</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>751</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="recursos_proyecto.qrc">:/LOGOS/Imagenes/uat.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>430</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="recursos_proyecto.qrc">:/LOGOS/Imagenes/pngegg.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>789</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="recursos_proyecto.qrc"/>
 </resources>
 <connections/>
</ui>
